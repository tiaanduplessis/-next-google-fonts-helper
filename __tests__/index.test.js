import React from 'react';
import NextGoogleFontsHelper from '../src';
import renderer from 'react-test-renderer';

test('<NextGoogleFontsHelper/>', () => {
  const component = renderer.create(
    <NextGoogleFontsHelper fonts={['Montserrat:wght@300;400;500;600']} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
