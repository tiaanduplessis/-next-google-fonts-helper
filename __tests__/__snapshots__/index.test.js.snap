// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NextGoogleFontsHelper/> 1`] = `
Array [
  <link
    crossOrigin={true}
    href="https://fonts.gstatic.com"
    rel="preconnect"
  />,
  <link
    as="style"
    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap"
    rel="preload"
  />,
  <link
    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap"
    media="print"
    onload="this.media='all'"
    rel="stylesheet"
  />,
  <noscript>
    <link
      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap"
      rel="stylesheet"
    />
  </noscript>,
]
`;
